src/algorithms/ctmrg.jl

```diff
@@ -96,11 +91,21 @@ function MPSKit.leading_boundary(envinit, state, alg::CTMRG)
     )
     env′, = ctmrg_iter(state, env, alg_fixed)
     envfix = gauge_fix(env, env′)
-    check_elementwise_convergence(env, envfix; atol=alg.tol^(3 / 4)) ||
+    check_elementwise_convergence(env, envfix; atol=alg.tol^(1 / 2)) ||
```

Comment:
do we really need this to be 1 / 2? is this an artifact from testing?

Reply:
the gaugefix tests use 1e-10 for the ctm and 1e-4 for the elementwise convergence so fairly close to 1/2. I never encountered problems with 1/2 but don't know if higher powers would also work